tests/Storage/Compression/Algorithms/XZTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Utopia\Tests\Compression\Algorithms;
```

Comment:
Would you please use:

```suggestion
namespace Tests\Storage\Compression\Algorithms;
```
to comply with psr4 autoload standards.